ImageSharpCompare/ImageSharpCompare.CalcDiffImageWithMask.cs

```diff
@@ -0,0 +1,269 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using SixLabors.ImageSharp.PixelFormats;
+
+using SixLabors.ImageSharp;
+
+namespace Codeuctivity.ImageSharpCompare;
+public static partial class ImageSharpCompare
```

Comment:
use  #pragma warning disable CA1724 // Type names should not match namespaces - this is accepted for now to prevent a breaking change public static partial class ImageSharpCompare #pragma warning restore CA1724 // Type names should not match namespaces - this is accepted for now to prevent a breaking change to silence CA1724 warnings